darwin.m

```diff
@@ -0,0 +1,67 @@
+// +build darwin
+
+#import <Foundation/NSArray.h>
```

Comment:
@fffw Why didn't you just use [elevate](https://github.com/getlantern/elevate)?  Aside from being ready already, elevate also doesn't require CGO.